### DISCUSSION TITLE
.NET (core) all versions: lack of fixes

### DISCUSSION THREAD

**Discussion by wstaelens** [dotnet/runtime#97630] [General]
Why is it that nobody fixes anything print related?

For the last weeks, months, actually years there have been issues reported related to printing. 
Even recently related to Windows 11 and Office. 

Almost **none** of these have been resolved.

Since the launch of .net core parts of the .NET Framework have not been ported related to printing. Pull requests have been made but the code was never added, completed or fixed. Some parts not being ported have [nothing todo with printing](https://github.com/dotnet/runtime/issues/51929#issuecomment-1164775635) but printing needs them (e.g. compression area).

**After 8 (or 10) years since the release of .net core these issues are still there**. 
Every release it is labeled as **Future**, postponed or whatever silly reason we heard over all these years.

All we see since Windows 8.1 are clumsy issues like this recently: https://www.windowslatest.com/2023/12/04/windows-update-accidentally-renames-all-printers-to-hp-m101-m106-on-windows-11-windows-10/  and nightmares for System Administrators and print fleet managers.

I'm well aware and I understand that printing is not sexy, has a low priority and that it requires you to invest in people and resources with little return compared to Azure and other projects.... But not spending any resources at fixing print issues in ± 3000 days, this is something we can't understand.

Why is it that nobody at Microsoft seems to care about printing, XPS and print related issues? In real life people open Word and print. They open Autocad and print, they open Excel and print,... every single day.

I tried to made it easier to gather print related issues together in runtime (but in reality it is a mix of wpf, runtime, winforms,...) https://github.com/dotnet/runtime/issues/78629 

What is needed to even get the team at Microsoft starting to fix these issues after 8 or 10 years? 
When can we expect to see some improvement?

**Comment by olowk** [comment]
We also have the feeling that microsoft is not taking print issues seriously. Only security issues are fixed, but else it seems that printing in windows is low low low priorty. Hope this will change soon.
